extensions/dicom-segmentation/src/getOHIFDicomSegSopClassHandler.js

```diff
@@ -127,6 +128,8 @@ export default function getSopClassHandlerModule({ servicesManager }) {
             []
           );
         }
+
+        refreshViewports();
```

Comment:
Why? a get function shouldn't generate any side effects like refreshing cornerstone viewports

Reply:
when opening http://localhost:3000/projects/idc-tcia/locations/us-central1/datasets/tcia-idc-datareviewcoordination/dicomStores/issue11-lung1-20201015-review/study/1.3.6.1.4.1.32722.99.99.203715003805996641695765332389135385095 could happen that the seg was not rendered since loadAndCacheDerivedDisplaySets() just load the seg (and all the react segmentation menu elements are still hidden, so they do not refresh the viewport). But you are right the refreshViewports should not be in the laod function of the displaySet, so I moved the refreshViewports into loadAndCacheDerivedDisplaySets function.